pkg/distro/rhel9/azure.go

```diff
@@ -572,35 +572,12 @@ var defaultAzureImageConfig = &distro.ImageConfig{
 }
 
 // Diff of the default Image Config compare to the `defaultAzureImageConfig`
+// The configuration for non-RHUI images does not touch the RHSM configuration at all.
+// https://issues.redhat.com/browse/COMPOSER-2157
 var defaultAzureByosImageConfig = &distro.ImageConfig{
 	GPGKeyFiles: []string{
 		"/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release",
 	},
-	RHSMConfig: map[subscription.RHSMStatus]*osbuild.RHSMStageOptions{
```

Comment:
I think this might need to be removed from `pkg/distro/rhel9/gce.go` too

Reply:
Done.